src/PostgREST/Auth.hs

```diff
@@ -39,16 +39,16 @@ data JWTAttempt = JWTInvalid JWTError
                 deriving (Eq, Show)
 
 {-|
-  Receives the JWT secret (from config) and a JWT and returns a map
+  Receives the JWT secret and audience (from config) and a JWT and returns a map
   of JWT claims.
 -}
-jwtClaims :: Maybe JWK -> BL.ByteString -> IO JWTAttempt
-jwtClaims _ "" = return $ JWTClaims M.empty
-jwtClaims secret payload =
+jwtClaims :: Maybe JWK -> Text -> BL.ByteString  -> IO JWTAttempt
```

Comment:
The audience argument should be `Maybe Text` rather than `Text` though, which will change our logic below slightly.

Reply:
I've moved the Maybe handling into the config parser and having it default to "", which simplifies the rest of the handling nicely.